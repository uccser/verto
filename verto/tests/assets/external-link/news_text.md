Check out this [news](news).
