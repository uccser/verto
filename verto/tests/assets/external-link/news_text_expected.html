<p>Check out this <a href="news">news</a>.</p>
